### DISCUSSION TITLE
Public GitHub Package couldn't found via GitHub Actions in the same Organisation

### DISCUSSION THREAD

**Discussion by jw2702** [community/community#24939] [Actions]
<p>Hi,</p>
<p>I recently started using GitHub Packages.  The GitHub repositories are publicly accessible and should therefore be easy to use.  Unfortunately, these are not found, for example in the IDE I am using, IntelliJ, but also not found in GitHub Actions.</p>
<p>GitHub Package: <a href="https://github.com/MineCodeDEV/Core/packages/577968" class="inline-onebox" rel="noopener nofollow ugc">Package dev.minecode.core.api · MineCodeDEV/Core · GitHub</a><br>
pom.xml: <a href="https://github.com/MineCodeDEV/Coins/blob/d40fe0d1297ebd93b6e22748e7e151aad2aa9596/api/pom.xml" class="inline-onebox" rel="noopener nofollow ugc">Coins/pom.xml at d40fe0d1297ebd93b6e22748e7e151aad2aa9596 · MineCodeDEV/Coins · GitHub</a></p>
<p>Does anyone know why the dependency can’t be found?</p>
<p>Regards<br>
Jerome</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Unfortunately even public Maven packages still require authentication for download. The “Authenticating to GitHub Packages” section in this document explains how to do that:</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/packages/guides/configuring-apache-maven-for-use-with-github-packages" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/packages/guides/configuring-apache-maven-for-use-with-github-packages" target="_blank" rel="noopener">Configuring Apache Maven for use with GitHub Packages - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/packages/guides/configuring-apache-maven-for-use-with-github-packages" target="_blank" rel="noopener">//docs.github.com/en/packages/guides/configuring-apache-maven-for-use-with-github-packages</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>AFAIK that applies to GitHub packages in general, only GHCR allows downloads of public images without any authentication.</p>

**Comment by jw2702** [comment]
<p>Hi,</p>
<p>Thank you for your answer.</p>
<p>However, I can also use, for example, <a href="https://github.com/orgs/SpongePowered/packages?repo_name=Configurate" rel="noopener nofollow ugc">Configurate</a> without authentication.  Was there an attitude made there that I can also make?</p>
<p>Regards<br>
Jerome</p>

**Comment by airtower-luna** [comment]
<p>If I’m not mistaken the snippets on the page for each package show how to add the dependency to the <code>pom.xml</code>. The “Authenticating to GitHub Packages” section of the page I linked describes how to configure access to the packages Maven repository in your Maven configuration, including token authentication. I haven’t worked with Maven too much, but my understanding is that you need both parts.</p>

**Comment by jw2702** [comment]
<p>Hi,</p>
<p>thanks again for your answer.</p>
<p>I’ve just authenticated myself and now everything works.</p>
<p>Regards<br>
Jerome</p>
